Sources/Network/TaxonomiesRequest.swift

```diff
@@ -30,6 +30,8 @@ struct TaxonomiesRequest: URLRequestConvertible {
             return route.rawValue + "/data/" + Endpoint.get
         case (.post, _):
             return Endpoint.post + route.rawValue
+        case (.get, _):
+            return Endpoint.get + "/data/" + route.rawValue
```

Comment:
Adding this case was the only way to allow the creation of URLrequests to succeed when `TaxonomyRequest.asURLRequest()` is called by any of the `refresh`{Taxonomy}`()` methods in `TaxonomiesService.downloadTaxonomies()`
Without that change, each of the taxonomy requests (except `refreshIngredientsAnalysisConfig()`) **always** fail.

The return value for the first case `(.get, .getIngredientsAnalysisConfig):`  never gets used by `refreshIngredientsAnalysisConfig()`. Instead, it uses `FilesRouter` to create its URL request. So it seems like that case should have been removed.

I also had to reorder parts of the return value for the case that I added, instead of  
`return route.rawValue + "/data/" + Endpoint.get`
I did
`return Endpoint.get + "/data/" + route.rawValue`
because that is how the actual URLs are formatted for the JSON data.

Reply:
@aleene, this is the change I was discussing with you last week. I believe that **none** of the Taxonomies were being downloaded prior to making this change. I know that you said: "I do not see why you would want an extra case, as you always want to know the configuration parameters," but cannot see what harm this change causes, and I really think it was necessary for downloading all the Taxonomies, including 'Label,' which I just added.

Reply:
I am 99% sure these changes were necessary because I believe that none of the Taxonomies were being downloaded prior to making this change.